dotnet/ComputerVision/RecognizeDomainSpecificContent/Program.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.CognitiveServices.Samples.ComputerVision.RecognizeDomainSpecificContent
+{
+    using Newtonsoft.Json.Linq;
+
+    class Program
+    {
+        public const string subscriptionKey = "<your training key here>"; //Insert your Cognitive Services subscription key here
+        public const string endpoint = "https://westus.api.cognitive.microsoft.com"; // You must use the same Azure region that you generated your subscription keys for.  Free trial subscription keys are generated in the westus region. 
+
+        static void Main(string[] args)
+        {
+            RecognizeDomainSpecificContentSample.RunAsync(endpoint, subscriptionKey).Wait(5000);
+
+            Console.WriteLine("\nPress ENTER to exit.");
+            Console.ReadLine();
+        }
+    }
+
+    public class RecognizeDomainSpecificContentSample
+    {
+        public static async Task RunAsync(string endpoint, string key)
+        {
+            Console.WriteLine("Recognize domain specific content (celebrities/landmarks) in images:");
+
+            string imageFilePath = @"Images\landmark.jpg"; // See this repo's readme.md for info on how to get these images. Alternatively, you can just set the path to any appropriate image on your machine.
+            string remoteImageUrl = "https://github.com/Azure-Samples/cognitive-services-sample-data-files/raw/master/ComputerVision/Images/celebrities.jpg";
+
+            await RecognizeDomainSpecificContentFromUrlAsync(imageFilePath, endpoint, key, "celebrities");
+            await RecognizeDomainSpecificContentFromStreamAsync(remoteImageUrl, endpoint, key, "landmarks");
+        }
+
+        static async Task RecognizeDomainSpecificContentFromStreamAsync(string imageFilePath, string endpoint, string subscriptionKey, string specificDomain)
+        {
+            if (!File.Exists(imageFilePath))
+            {
+                Console.WriteLine("\nInvalid file path");
+                return;
+            }
+            try
+            {
+                HttpClient client = new HttpClient();
+
+                // Request headers.
+                client.DefaultRequestHeaders.Add("Ocp-Apim-Subscription-Key", subscriptionKey);
+                string uri = endpoint + @"/vision/v2.0/models/"+specificDomain+ @"/analyze";
```

Comment:
Lines like this look better as 
string uri = $"{endpoint}/vision/v2.0/models/{specificDomain}/analyze"